### ISSUE TITLE
TypeScript types

### ISSUE THREAD

**Issue by torkelrogstad** [altangent/lnd-async#7]
There's no typings for this package. Is that something that would be desired? If so, I could take a stab at implementing them.s 

I understand that the methods exposed after `lnd.connect()` aren't implemented in this package, and that this package is just really a somewhat thin wrapper around gRPC calls. However, I believe it is possible to write types for the gRPC calls, meaning that it would be possible to interact with `lnd` API calls, and have the TypeScript compiler/your IDE tell you what the response contains.

**Comment by bmancini55** [comment]
Absolutely, I think that would be a great addition.  Especially if the gRPC protocol definitions can be mapped out.

**Comment by torkelrogstad** [comment]
I've actually started on writing out the gRPC types, but it's quite tedious. Is there some tool out there for extracting it from proto files? It probably won't be 100% correct, but it would be a good starting point.
